data/hai/hai reveal 5 philosophers.txt

```diff
@@ -415,19 +508,35 @@ mission "Hai Reveal [B06-long] Tarazed Analysis"
 			label lead
 			`	You clarify that technically it's a lead on cases that match the same profile as his wife's, not strictly his wife's case. His expression falters momentarily and then he seems to find some internal resolve to draw upon. "Good enough for me... Thank you, for your honesty."`
```

Comment:
I wonder if it's possible to contact Goraz's wife on Elenchus. Maybe, depending on what actions you take, you could get her back to him yourself, or she might come back on another transport (which means he'd send you a message), or she'd refuse and stay on Elenchus.

Reply:
That belongs in a follow-up mission. Two human ships with jump drives are still at Elenchus (presumably with a massive rental payment) so they can take people back and forth. I expect Elenchus also has a few jump drives in reserve.

Reply:
We need some post-campaign "check on your friends" missions, and Goraz should certainly be one of them.